staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -407,8 +407,8 @@ func (a *Admission) EvaluatePod(ctx context.Context, nsPolicy api.Policy, nsPoli
 		klog.V(2).InfoS("failed to parse PodSecurity namespace labels", "err", nsPolicyErr)
 		auditAnnotations["error"] = fmt.Sprintf("Failed to parse policy: %v", nsPolicyErr)
 	}
-	// TODO: log nsPolicy evaluation with context (op, resource, namespace, name) for the request.
 
+	klog.V(2).InfoS("nsPolicy evaluation", "op", attrs.GetOperation(), "resource", attrs.GetResource(), "namespace", attrs.GetNamespace(), "name", attrs.GetName())
```

Comment:
A couple thoughts on this:
- This is going to be very verbose. Let's drop it to a lower verbosity, and use the `Enabled` check to avoid constructing arguments when disabled.
- We should log the policy that was evaluated.

```suggestion
	if klog.V(4).Enabled() {
		klog.InfoS("Pod Security evaluation", "policy", nsPolicy.String(), "op", attrs.GetOperation(), "resource", attrs.GetResource(), "namespace", attrs.GetNamespace(), "name", attrs.GetName())
	}
```

Reply:
The policy have no String() function to call. So I just display the nsPolicy, just like 
`klog.InfoS("Pod Security evaluation", "policy", nsPolicy)`
And a small test about the log will show:
```
I0925 09:32:21.623597    7388 main.go:48] "test" policy={Enforce:{Level:1 Version:{major:1 minor:2 latest:false}} Audit:{Level:2 Version:{major:2 minor:1 latest:false}} Warn:{Level:3 Version:{major:3 minor:2 latest:false}}}
```

Reply:
That's pretty verbose... I'd prefer to use the `LevelVersion.String()` method. You could just inline `fmt.Sprintf("%v", nsPolicy)`, or you could add a string method to Policy that does just that.

Reply:
Done, PTAL